screens/ClerkLoginScreen.js

```diff
@@ -1,26 +1,25 @@
-import React from "react";
+import React from 'react';
 import {
   AsyncStorage,
   Button,
-  Picker,
-  View,
   Keyboard,
-  TouchableWithoutFeedback
-} from "react-native";
-
-import { BASE } from "../lib/common";
-import { styles, TextInput } from "../styles";
+  Picker,
+  TouchableWithoutFeedback,
+  View
+} from 'react-native';
+import BASE from '../lib/common';
```

Comment:
The only reason I had to update this was because the old `common.js` did the export of `BASE` differently; but the `prettier` script prettiers the whole thing 😅